core/src/main/java/com/novoda/noplayer/internal/mediaplayer/AndroidMediaPlayerFacade.java

```diff
@@ -38,6 +40,7 @@
     private int currentBufferPercentage;
 
     private MediaPlayerCreator mediaPlayerCreator;
+    private ContentType contentType;
```

Comment:
Can this be `@Nullable`? and if can, shall we protect methods in the class for that case?

Reply:
Good point, the `MediaPlayer` should be too 🤔

Reply:
dc6ee27